packages/core/test/helpers/test-utils.ts

```diff
@@ -6,6 +6,6 @@ export function sleep(ms = 0) {
   });
 }
 
-export function resolveFromRoot(...pathSegments: string[]) {
-  return resolve(__dirname, '..', '..', ...pathSegments);
+export function resolveFromRoot(...pathSegments: string[]): string {
```

Comment:
resolve doesn't return string by default?

Reply:
Yes it does, I wanted to make it a bit more explicit I guess.

Reply:
In my experience declaring return type can be harmful and dangerous. This time it is a simple one liner with return, but even in this case people can break it... believe me...